src/sentry/conf/server.py

```diff
@@ -851,11 +851,11 @@ def create_partitioned_queues(name):
     # Enable the relay functionality, for use with sentry semaphore. See
     # https://github.com/getsentry/semaphore.
     'organizations:relay': False,
-    # DEPCREATED: pending removal.
+    # Enable require 2FA across organization
```

Comment:
Damn, not sure why I marked this feature as deprecated 🤔.  I assume this was me.

Reply:
Oh it was from a while ago. Forgot to update this earlier